bindings/nodejs/lib/types/models/info/node-info-protocol.ts

```diff
@@ -54,15 +53,15 @@ export interface ProtocolParameters {
     /**
      * Current supply of base token.
      */
-    tokenSupply: u64;
+    tokenSupply: string;
```

Comment:
I am not sure at all :trollface:

Reply:
seems like it's not trivial to achieve that. One direction BigInt->String is straightforward, but for the other direction there doesn't seem to be an easy way. One would have to convert the fields to bigint manually, right? Also ... this is coming from a `.json` file not `get-info` which means the solution needs to be generic over different sources?

Reply:
> direction there doesn't seem to be an easy way. One would have to convert the fields to bigint manually, right?

Yes, that's what we did in the other functions that require this conversion
The .json file should have the same format as how get-info will be converted to if there is a bigint, so it will just not convert anything and be correct if you use it as parameter somewhere

Reply:
Fixed by @kwek20 :pray: